src/algorithms/bubbleSort.spec.js

```diff
@@ -0,0 +1,69 @@
+import { bubbleSortAlt, bubbleSort } from './bubbleSort.js';
+
+describe('bubbleSort', () => {
+    it('should sort array and return [-6, 0, 1, 4, 7]', () => {
```

Comment:
bubbleSort doesn't return array, here and below

Reply:
discussed, fixed